serving-sdk/src/test/java/software/amazon/ai/mms/servingsdk/ModelServerEndpointTest.java

```diff
@@ -13,39 +13,115 @@
 
 package software.amazon.ai.mms.servingsdk;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
+
+import org.junit.Assert;
```

Comment:
still from junit?
Should this be testng?

Reply:
I moved to junit completely.